### ISSUE TITLE
Putting images side-by-side

### ISSUE THREAD

**Issue by Lenticulostriate** [Arthur-Milchior/anki-resize-image#26]
This add-on is a godsend but I can no longer put images next to one another. Any time I try to paste an image it will automatically upload the images below the pre-exisiting image. How can I fix this?

![image](https://user-images.githubusercontent.com/66183413/83819109-a579a100-a68e-11ea-9454-af353492fa10.png)

**Comment by thiswillbeyourgithub** [comment]
I've been having issues with this quite a lot but it's quite complicated to fix I think.

What I do, and what you can do, is ctrl+shift+x (or find the adequate button in the editing window buttons) to display the HTML code of the card. There you can do what you want. \<br> means "line break" and \<div> are brackets that surround a sentence that is equivalent to \<span> except that ending a div bracket creates a newline whereas ending a span does not.

So basically, look where there is a br or div and edit it.

Also, please note : there is not undo button with html edition, so open the html window, copy it into the clipboard, then do your edition, close the window, then you can paste it again if you messed up the code.

I recommend rearranging pictures like this only after doing most of the editing of the card, as it can sometimes create wonky behavior when further editing.

**Comment by thiswillbeyourgithub** [comment]
actually : there is an addon code "remove linebreaks" that might just do the trick.

https://ankiweb.net/shared/info/2028179508

**Comment by Lenticulostriate** [comment]
Awesome, Ill try the add-on out and see how it goes.

**Comment by Lenticulostriate** [comment]
It worked!!! Thank you!!

**Comment by Arthur-Milchior** [comment]
Thank you @thiswillbeyourgithub , I didn't even know this.
I really don't know CSS/HTML well enough to deal with all of those real life problems :s
